src/api/app/components/workflow_run_detail_component.html.haml

```diff
@@ -22,6 +26,24 @@
     %h5 Webhook Request Payload
     %pre.border.p-2#request-payload
       = pretty_request_payload
+  .tab-pane.fade{ id: "scm-reports-tab-content#{id}", role: 'tabpanel',
+                  aria: { labelledby: "scm-reports-tab#{id}" } }
+    - if status_reports.any?
+      .list-group.list-group-flush
+        - status_reports.each do |status_report|
+          .list-group-item
+            %h6 Response body
+            %p
+              %span.badge{ class: status_report.status_class }= status_report.status.humanize
```

Comment:
I tested with GitHub and had cases where `status_report.status` is `nil`. This explodes when calling `humanize` on it

Reply:
Seems the default value on the `status` column doesn't have an effect

Reply:
okay, havent realized that the migration changed meanwhile... my DB was outdated